x/concentrated-liquidity/incentives_test.go

```diff
@@ -203,6 +206,13 @@ func chargeIncentive(incentiveRecord types.IncentiveRecord, timeElapsed time.Dur
 	return incentiveRecord
 }
 
+// emittedIncentives returns the amount of incentives emitted by incentiveRecord over timeElapsed.
+// It uses the same logic as chargeIncentive for calculating this amount.
+func emittedIncentives(incentiveRecord types.IncentiveRecord, timeElapsed time.Duration) sdk.Dec {
```

Comment:
Note to reviewer: this test helper is unused in this PR, but it is an important abstraction to have and is used heavily in subsequent PRs relating to incentives